### ISSUE TITLE
pytorch_squeezenet.ipynb training image transformer (line 8)

### ISSUE THREAD

**Issue by abelavit** [alok-ai-lab/pyDeepInsight#36]
Dear Author,

I am using the pyDeepInsight package for my protein task and closely following the example https://github.com/alok-ai-lab/pyDeepInsight/blob/master/examples/pytorch_squeezenet.ipynb. I have a question on the training of the image transformer on training data. I wonder if the training (fit) should be taking place on the normalised training data (X_train_norm), and not the original training data (X_train). Below is the code snippet from the example at line 8:

`it.fit(X_train, y=y_train, plot=True)`
`X_train_img = it.transform(X_train_norm)`
`X_test_img = it.transform(X_test_norm)`

Thank you for your assistance.

**Comment by kaboroevich** [comment]
Hello Abel,

You are correct that it using X_train_norm is the more sensible choice. When this example was originally written (in 2020), I was getting strange results from running sklearn's t-SNE on the normalized data, which was producing an image with many overlapping features in the centre due to many spread out features. This was very different than the t-SNE results that I observed in MatLab and R (see image below). However, running it the non-normalized data produced a much better spread of the features similar to MatLab and R. I never determined why this occurred, but it appears to longer be an issue. I will update the example to use the normalized data. Thank you for noticing this.

![image](https://github.com/alok-ai-lab/pyDeepInsight/assets/872618/7891982e-a8f7-4832-b1a9-dab4eb1f0e12)
